### PR TITLE
Fix sphinx3 deprecation

### PR DESCRIPTION
**Pull request by Daltz333**

### CODE REVIEW

**Comment by humitos** [comment]
Hi @Daltz333! Thanks for your contribution. Could you take a look at the tests? They are failing for some reason.

**Comment by Daltz333** [comment]
Looks like it's passing now. I added an import and check to maintain Sphinx<2 compatibility.

https://travis-ci.org/github/readthedocs/sphinx-notfound-page/builds/666406632?utm_medium=notification&utm_source=github_status

**Comment by Daltz333** [comment]
@humitos any objections?

File: notfound/utils.py [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import docutils
 import re
-from sphinx.builders.html import DirectoryHTMLBuilder
+import sphinx
+
+# Sphinx <2 Compatibility
+if int(sphinx.__version__[0]) >= 2:
```
Comment (humitos) [inline notfound/utils.py]: Please, use `sphinx.version_info` here as we are already doing at https://github.com/readthedocs/sphinx-notfound-page/blob/master/notfound/extension.py#L222

**Comment by Daltz333** [comment]
@humitos Addressed!

**Pull request merged**